doc/whats_new/upcoming_changes/README.md

```diff
@@ -22,7 +22,8 @@ This file needs to be added to the right folder like `sklearn.linear_model` or
 `sklearn.tree` depending on which part of scikit-learn your PR changes. There
 are also a few folders for some topics like `array-api`, `metadata-routing` or `security`.
 
-In almost all cases, your fragment should be formatted as a bullet point.
+In almost all cases, your fragment should be formatted as a **single** bullet point.
+Note the aggregation software cannot handle more than one bullet point per entry.
```

Comment:
Not sure that this second line is necessary. Maybe just adding 'single' above is adequate?